### ISSUE TITLE
Check more than one role in route

### ISSUE THREAD

**Issue by EzeRangel** [appzcoder/laravel-admin#35]
I was trying the package and came with a need to check more than one role in a route. Digging in the `HasRoles` trait you only check for a string in the role so I extended the code to check an array.

Now you can do something like this.

```
$api->group(["middleware" => "roles", "roles" => ["admin", "user"] ], function($api){
...
```

In HasRoles add this.
```
public function hasRole($role)
    {
        if (is_string($role)) {
            return $this->roles->contains('name', $role);
        }

        //In case we have more than one role to check.
        if(is_array($role)){
            foreach($role as $r){
                if($this->hasRole($r)){
                    return true;
                }
            }
        }

        return !!$role->intersect($this->roles)->count();
    }
```

Hope it helps!

**Comment by sohelamin** [comment]
@EzeRangel Can you make a PR for this?

**Comment by sohelamin** [comment]
Fixed at ddce649f9
